### ISSUE TITLE
Another tool

### ISSUE THREAD

**Issue by Tejpbit** [apiraino/autorandr#2]
>If after reading this guide you think that I just wasted my time because there's a tool that does this automagically, please let me know, I'll be super-happy to try it! :-)

Hey! Don't know if this is still relevant.
First, you haven't wasted your time. You've probably learnt some stuff along the way. :1st_place_medal: 
Anyhow, I recently found another tool called [srandrd](https://github.com/jceb/srandrd).
Apart from installing, it requires ~~a handler-script and~~ a configuration file for common monitors with a default case for new ones.
Ironically it stopped working for me just now (probably an unrelated issue as I hadn't rebooted since installing it) and while searching around for my problem I found your repo.

**Comment by apiraino** [comment]
Hi!

thanks for mentioning that tool, that looks really cool and probably would have used it at the time!

The only difference AFAICT is that `srandrd` does poll continuously the monitor situation and triggers a helper script, while I priviledged the `udev` approach to be notified "passively" of something changing, I thought that it was less "wasteful".

Anyhow, I just dropped the whole thing (along Lubuntu) because other Ubuntu flavours just work out of the box, so like you said, it was fun because I learned something interesting ;)

**Comment by holocronweaver** [comment]
Which Ubuntu flavor + GPU driver are you using? Because plain Ubuntu 16.04 + Nvidia still requires a tool like autorandr afaict. If I am mistaken, please correct me!

**Comment by apiraino** [comment]
@holocronweaver at the time of writing of the tool I was using Lubuntu 16.xx. The distro window manager (LXDE) is lacking a tool to manage the screen addition/removal, hence I hacked together this tool to script `arandr`. The laptop was a Dell XPS 13 with just a plain boring Intel chipset, no discrete graphic card.

Now I moved to a new laptop and installed Ubuntu + Gnome. The window manager is able to manage out of the box this transition smoothly. That's also what my Ubuntu-powered friends say, but YMMV.
